terminal/TerminalServer.cpp

```diff
@@ -358,7 +427,120 @@ void startUserTerminal() {
   uth.run();
 }
 
-int main(int argc, char** argv) {
+void startJumpHostClient() {
+  string idpasskey = FLAGS_idpasskey;
```

Comment:
in startUserTerminal we handle either idpasskeyfile or commandline specified idpasskey.  Let's move that logic into main.  So, in main, if the idpasskeyfile option is set, we load the passkey from file and set FLAGS_idpasskey.  Then this function doesn't have to change and we don't have to check for FLAGS_idpasskeyfile before starting startUserTerminal